docs/backuprestore-with-oadp.md

```diff
@@ -28,6 +28,42 @@ anntotations:
 If the annotation is provided in the backup or restore CRs, they will be applied in increasing order based on the annotation value. The LCA will move on the next group of CRs only after completing the previous group with the same wave number.
 If no `lca.openshift.io/apply-wave` annotation is defined in the backup or restore CRs, which means no particular order is required, they will be applied all together. 
 
+## LCA apply label annotation
+
+OADP backup doesn't support backing up specific CRs by object names, and the way to scope/filter backup specific resources is by using label selector.
+
+This annotation provides a way for users to pass in the specific resources, and LCA will apply the label to those resources internally in order for the backup CR to only pick them up.
+
+The resources could be passed using the
+`lca.openshift.io/apply-label` annotation. The value should be a list of comma separated objects in `group/version/resource/name` format for cluster-scoped resources or `group/version/resource/namespace/name` format for namespace-scoped resources, and it should be attached to the related Backup CR. For example:
+
+```
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  name: acm-klusterlet
+  namespace: openshift-adp
+annotations:
+  lca.openshift.io/apply-label: rbac.authorization.k8s.io/v1/clusterroles/klusterlet,apps/v1/deployments/open-cluster-management-agent/klusterlet
+labels:
+  velero.io/storage-location: default
+spec:
+  includeNamespace:
+   - open-cluster-management-agent
+  includeClusterScopedResources:
+   - clusterroles
+  includeNamespaceScopedResources:
+   - deployments
+```
+
+By defining this annotation, ther user indicates a desire to scope to those resources only. LCA will parse the annotations, and apply a label, called `lca.openshift.io/backup: true` to those resources. LCA then adds this labelSelector when creating the backup CR:
```

Comment:
nit: can you fix the sentence here `ther user indicates a desire to scope to those resources only`

Reply:
thanks, fixed